changelogs/unreleased/gh-9979-space-format-object.md

```diff
@@ -0,0 +1,4 @@
+## feature/box
```

Comment:
> Implementation note:
The format_object method was not implemented for net.box spaces:
>1. Remote verification will require sending tuples over the network;
>2. May fail when format references collations or types unavailable
on client.

1. `format_object` is not a method.
2. Let's remove this `Implementation note` line (the whole commit message is an implementation note, actually). And let's put this information before test adjustments.
3. "May fail when format references collations or types unavailable on client." - It's not clear, what may fail.